docs/api/en/objects/BatchedMesh.html

```diff
@@ -222,22 +224,35 @@ <h3>
 			the length of the given geometry index buffer.
 		</p>
 		<p>
-			Adds the given geometry to the [name] and returns the associated index referring to it.
+			Adds the given geometry to the [name] and returns the associated geometry id referring to it to be used in other functions.
+		</p>
+
+		<h3>
+			[method:Integer addInstance]( [param:Integer geometryId] )
+		</h3>
+		<p>
+			[page:Integer geometryId]: The id of a previously added geometry via "addGeometry" to add into the [name] to render. 
+		</p>
+		<p>
+			Adds a new instance to the [name] using the geometry of the given geometryId and returns a new id referring to the new instance to be used
+			by other functions.
 		</p>
 
 		<h3>
-			[method:Integer setGeometryAt]( [param:Integer index], [param:BufferGeometry geometry] )
+			[method:Integer setGeometryAt]( [param:Integer geometryId], [param:BufferGeometry geometry] )
 		</h3>
 		<p>
-			[page:Integer index]: Which geometry index to replace with this geometry.
+			[page:Integer geometryId]: Which geometry id to replace with this geometry.
 		</p>
 		<p>
-			[page:BufferGeometry geometry]: The geometry to substitute at the given geometry index.
+			[page:BufferGeometry geometry]: The geometry to substitute at the given geometry id.
 		</p>
 		<p>
-			Replaces the geometry at `index` with the provided geometry. Throws an error if there is not enough space reserved for geometry at the index.
+			Replaces the geometry at `geometryId` with the provided geometry. Throws an error if there is not enough space reserved for geometry.
+			Calling this will change all instances that are rendering that geometry.
```

Comment:
No changes here, but perhaps the future API is:

- `setGeometryAt( geometryId, geometry )` overwrites a geometry (up to the allocated vertex count), and affects all instances of that geometry
- `setInstanceAt( instanceId, geometryId )` changes the geometry instantiated by a specific instance ID.